README.md

```diff
@@ -1,2 +1,5 @@
-# gamepad-navigator
-GSoC 2020 project
+# Gamepad Navigator
+
+Gamepads are popular devices, and a lot of work has been done to extend its usefulness to a wide variety of people. Some of the game controllers, such as the [Xbox Adaptive controller](https://www.microsoft.com/en-us/p/xbox-adaptive-controller/8nsdbhz1n3d8?rtc=1), the [Logitech G Adaptive Gaming Kit](https://www.logitechg.com/en-us/products/gamepads/adaptive-gaming-kit-accessories.html), and the [one-handed controllers](https://www.evilcontrollers.com/ps4-one-handed-controller), strive to include a broader range of people in traditional PC and console gaming.
```

Comment:
This should fail to lint because of the line lengthh, which suggests to me that the list of files checked is not complete.